### ISSUE TITLE
Tutorial 4: UI examples: confirm snarkyjs is rebranded 

### ISSUE THREAD

**Issue by barriebyron** [o1-labs/docs2#623]
After I moved Tutorial 9 examples from a private repo to the /docs2 repo in https://github.com/o1-labs/docs2/pull/621 we noticed that we still needed to rebrand snarkyjs > o1js in the example code

I did a search for the docs repo and noticed a few files in Tutorial 4 that showed instances of snarkyjs (but maybe these files are only on my computer?)

let's look at these files [snarkyjs in tuto 4 search](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN2dUQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--47de89380e5b5123d8e71680bb94a27fc4791b59/image.png)

**Comment by microbecode** [comment]
At least tutorial 5 still has 'snarkyjs' references.
Maybe I could just write a PR to rename all of those?

**Comment by barriebyron** [comment]
@microbecode if you are able to successfully run the tutorial with o1js, that is the time to submit a PR to change those instances of product and version the tutorial is confirmed working with.

**Comment by microbecode** [comment]
Good point that its functionality should be confirmed. Not so straightforward. Let's see if I'd fix some which I happen to use.
